dev/tidb-in-kubernetes/troubleshoot.md

```diff
@@ -266,3 +266,69 @@ If the cluster is successfully deployed, check the network using the following s
     - Check whether the relevant route rules on the node are correct.
     - Check whether the network plugin service works well.
     - Refer to [network connection failure between Pods](#network-connection-failure-between-pods) section.
+
+## TiKV Store is in `Tombstone` status abnormally
+
+Normally for each running Pod, there is a corresponding TiKV store in UP status. However, concurrent scale-in or scale-out on TiKV components might cause part of TiKV stores to fall into `Tombstone` status abnormally. If this happens, try the following steps to fix it:
```

Comment:
"In most cases" is not accurate - because it indicates uncertainty and exceptions. Here it's describing a normal state, which is "normally" certain :)